browser-extension/manifest.json

```diff
@@ -61,6 +61,12 @@
       "js": ["fetchTournesolToken.js"],
       "run_at": "document_end",
       "all_frames": true
+    },
+    {
+      "matches": ["https://tournesol.app/*"],
+      "js": ["fetchTournesolRecommendationsLanguages.js"],
```

Comment:
Is there a reason to not put this script in the same array as `fetchTournesolToken.js`? They are both triggered at `document_end` with the same `all_frames` parameter.

Reply:
No, I just forgot it was possible. I merged them.

Actually we could run it at `document_start` but it probably wouldn't be enough to solve the timing issue mentioned above.